Maple2.Model/Metadata/ItemMetadata.cs

```diff
@@ -97,3 +100,11 @@ public record ItemMetadataHousing(
     int TrophyId,
     int TrophyLevel,
     int InteriorLevel);
+
+public record DefaultHairMetadata(
```

Comment:
What do you think about setting default values to simplify construction when parsing:
```cs
public record DefaultHairMetadata(
    Vector3 BackPosition = default,
    Vector3 BackRotation = default,
    Vector3 FrontPosition = default,
    Vector3 FrontRotation = default,
    float MinScale = 0f,
    float MaxScale = 1f);
```

This way you can omit fields you don't want to set.